tests/AbstractLoopTest.php

```diff
@@ -398,6 +398,35 @@ function () {
         $this->loop->run();
     }
 
+    public function testSignal()
+    {
+        if (!extension_loaded('pcntl')) {
+            $this->markTestSkipped('Signal test skipped because PCNTL extension isn\'t loaded.');
```

Comment:
Another way around the support of `PCNTL` would be an extra method that could be called to check if the loop supports signals. Thoughts?